WooCommerce/Classes/POS/ViewModels/TotalsViewModel.swift

```diff
@@ -111,6 +111,9 @@ final class TotalsViewModel: ObservableObject, TotalsViewModelProtocol {
 
     func startSyncingOrder(with cartItems: [CartItem], allItems: [POSItem]) {
         guard CartItem.areOrderAndCartDifferent(order: order, cartItems: cartItems) else {
+            Task { @MainActor in
+                await prepareConnectedReaderForPayment()
```

Comment:
If we repeat Check Out with the same order items and the reader is connected, we expect the payment to be restarted automatically.

Reply:
We also need to make payment automatically in Checkout when the reader is connected but I will leave it for another task

https://github.com/woocommerce/woocommerce-ios/issues/13431